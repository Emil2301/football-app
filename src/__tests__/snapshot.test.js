import React from "react";
import { create } from "react-test-renderer";
import Error from "../components/Error";

describe("Error component", () => {
  test("it matches the snapshot", () => {
    const component = create(<Error />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});