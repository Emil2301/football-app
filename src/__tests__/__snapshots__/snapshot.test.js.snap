// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component it matches the snapshot 1`] = `
<div>
  <p>
    Error: Path does not exist
  </p>
</div>
`;
